src/Command/Ide/IdeOpenCommand.php

```diff
@@ -20,8 +21,8 @@ class IdeOpenCommand extends IdeCommandBase {
    */
   protected function configure() {
     $this->setDescription('Open a Cloud IDE in your browser')
-    ->addOption('cloud-app-uuid', 'uuid', InputOption::VALUE_REQUIRED, 'The UUID of the associated Cloud Platform Application')
-    ->setHidden(AcquiaDrupalEnvironmentDetector::isAhIdeEnv());
+      ->addArgument('applicationUuid', InputArgument::OPTIONAL, 'The UUID or alias of the associated Cloud Platform Application')
```

Comment:
I wonder if we should instead have an optional argument for the Cloud IDE UUID here.